pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -409,6 +409,11 @@ func (gce *GCECloud) ScrubDNS(nameservers, searches []string) (nsOut, srchOut []
 	return nameservers, srchOut
 }
 
+// HasClusterID returns true if the cluster has a clusterID
+func (gce *GCECloud) HasClusterID() bool {
```

Comment:
I could rather do that cleanup as a part of a separate PR.  It really isn't related to this PR.